src/components/dialogs/notificationDialogUtils/NotificationIndividualMergeCompleteDialog.jsx

```diff
@@ -8,12 +11,23 @@ export default function NotificationIndividualMergeCompleteDialog({
   onClose,
   notification,
 }) {
+  const history = useHistory();
   const notificationType = notification?.message_type;
   const currentNotificationSchema = get(
     notificationSchema,
     notificationType,
   );
-  const availableButtons = [];
+  const mergedIndividualGuid =
+    notification?.message_values?.target_individual_guid;
+  const onClickView = () =>
+    history.push(`/individuals/${mergedIndividualGuid}`);
```

Comment:
This should probably be a link. That might require too many changes at this point though.

Reply:
I dont think so... I think you could just do 
```
  const availableButtons = [
    {
      name: 'view',
      buttonId: 'VIEW_INDIVIDUAL',
      href: `/individuals/${mergedIndividualGuid}`
    },
  ];
```
then in NotificationDetailsDialog render a button if it's on click, or a buttonlink if it's href

Reply:
by the way, one great reason to try to limit history.push is that screen readers dont understand them, and when a user hovers over the link to try to see the URL it wont work. you also cant right click and open in a new tab, etc..

Reply:
Done